src/froentend/pages/home/Home.jsx

```diff
@@ -5,6 +5,9 @@ function Home() {
 	return (
 		<div>
 			<Navbar />
+			<div style={{ "margin-top": "8rem", textAlign: "center" }}>
```

Comment:
Avoid inline styling

Reply:
yes i have used inline styling just for Preview Purpose I will remove it Later

Reply:
Oh, then it's alright ^_^